### DISCUSSION TITLE
Best way to know whereas the app is running on Linux

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#11550] [Q&A] [answered]
What is the best way to detect whereas an app is running on Linux?

A. `if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))`

B. `if (Environment.OSVersion.Platform == PlatformID.Unix)`

**Comment by jeromelaban** [comment, marked as answer]
Both are valid, but A is commonly used (this is also what MSBuild intrinstics use).
